polytiles/polytiles.py

```diff
@@ -23,30 +23,28 @@
 except ImportError:
     HAS_SQLITE = False
 
-DEG_TO_RAD = pi/180
-RAD_TO_DEG = 180/pi
+DEG_TO_RAD = pi / 180
+RAD_TO_DEG = 180 / pi
 TILE_SIZE = 256
 LIST_QUEUE_LENGTH = 32
 
 
 def box(x1, y1, x2, y2):
     return Polygon([(x1, y1), (x2, y1), (x2, y2), (x1, y2)])
 
-
```

Comment:
[Per PEP8](https://pep8.org/#blank-lines), there should be 2 blank lines between functions. Please run a PEP8 validator on the file.